packages/primitives/popover/src/popover-content-attributes.component.ts

```diff
@@ -0,0 +1,198 @@
+import { Component, computed, forwardRef } from '@angular/core';
+import { RdxPopoverContentAttributesToken } from './popover-content-attributes.token';
+import { injectPopoverRoot } from './popover-root.inject';
+import { RdxPopoverAnimationStatus, RdxPopoverState } from './popover.types';
+import { isRdxPopoverDevMode } from './popover.utils';
+
+@Component({
+    selector: '[rdxPopoverContentAttributes]',
+    standalone: true,
+    template: `
+        <ng-content />
+    `,
+    styles: `
```

Comment:
@pimenovoleg you mean this stuff is unnecessary? Totally styleless solution, am I right?

Reply:
Yes true, lets be headless :)

Reply:
Ok, get it. I can't see any room for CSS variables so far, are they usefull for `popover`? Any suggestions?